main.go

```diff
@@ -23,6 +23,21 @@ import (
 
 var Version = ""
 
+var Logo = `
```

Comment:
If this is not expected to change:

```suggestion
const Logo = `
```

Reply:
Oh! Nice :)